data/ledger.json

```diff
@@ -13775,11 +13775,29 @@
+=======
+>>>>>>> 1259828 (rebased ledger as per docs after I negelected to commit the last calc)
```

Comment:
:eyes:

Reply:
Whoops, missed that one

Reply:
No biggie, just editing the "ledger" 😅

Reply:
> No biggie, just editing the "ledger" sweat_smile

what could possibly go rong? 😄 I appreciate your intervention & 👀  @alalonde & @polimyl 🙏🏻